src/controller/python/chip/utils/CommissioningBuildingBlocks.py

```diff
@@ -155,7 +155,7 @@ async def AddNOCForNewFabricFromExisting(commissionerDevCtrl, newFabricDevCtrl,
 
     await commissionerDevCtrl.SendCommand(existingNodeId, 0, opCreds.Commands.AddTrustedRootCertificate(chainForAddNOC.rcacBytes))
     resp = await commissionerDevCtrl.SendCommand(existingNodeId, 0, opCreds.Commands.AddNOC(chainForAddNOC.nocBytes, chainForAddNOC.icacBytes, chainForAddNOC.ipkBytes, newFabricDevCtrl.nodeId, 0xFFF1))
-    if resp.statusCode is not opCreds.Enums.OperationalCertStatus.kSuccess:
+    if resp.statusCode is not opCreds.Enums.NodeOperationalCertStatusEnum.kOk:
```

Comment:
Same as above, should this be **kOK**?

Reply:
This is a decision the people maintaining the Python APIs need to make.  I filed https://github.com/project-chip/connectedhomeip/issues/24486 to decide that.